src/pages/token/rewards/RewardsPage.tsx

```diff
@@ -36,13 +44,30 @@ const RewardsPage = () => {
 
   const { complianceState } = useComplianceState();
   const { isTablet, isNotTablet } = useBreakpoints();
-
+  const { usdcDenom } = useTokenConfigs();
+  const usdcDecimals = 24; // hardcoded solution; fix in OTE-390
   const { enableStaking, tradingRewardsRehaul: tradingRewardsRehaulEnabled } = testFlags;
 
+  const stakingRewards: AccountBalance[] =
+    useAppSelector(getStakingRewards, shallowEqual)?.totalRewards?.toArray() ?? [];
+
+  const totalUsdcRewards = stakingRewards?.reduce((total: number, reward) => {
+    if (reward?.denom === usdcDenom && reward.amount) {
+      return total + parseFloat(reward.amount);
+    }
+    return total;
+  }, 0);
+
   const showMigratePanel = import.meta.env.VITE_V3_TOKEN_ADDRESS && isNotTablet;
   const showGeoblockedPanel =
     tradingRewardsRehaulEnabled && complianceState !== ComplianceStates.FULL_ACCESS;
+  const showStakingRewardPanel = totalUsdcRewards > 0 && !showGeoblockedPanel && enableStaking;
 
+  const stakingRewardPanel = (
+    <StakingRewardPanel
+      usdcRewards={MustBigNumber(totalUsdcRewards).div(Number(`1e${usdcDecimals}`))}
```

Comment:
nit: you can use `formatUnits` to do this.

Reply:
ooh ok will follow up with this in my merge all flags pr